Tests/FunctionalTests/Sources/FTRSwiftTests.swift

```diff
@@ -69,6 +69,26 @@ class FunctionalTestRigSwiftTests: XCTestCase {
         .assertWithMatcher(grey_nil())
   }
 
+  func testInteractionWithALabelWithParentHidden() {
+    let checkHiddenBlock:GREYActionBlock =
+        GREYActionBlock.actionWithName("checkHiddenBlock", performBlock: { element, errorOrNil in
+                                        // Check if the found element is hidden or not.
+                                        let superView:UIView! = element as! UIView
```

Comment:
This code refers to ObjC. While we don't have an official style guide, I'd suggest keeping both styles in.